### PR TITLE
Update jing.rb

### PR DESCRIPTION
**Pull request by sachinsr099**
- [Y ] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document? 
- [Y ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change? 
- [N ] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
  P.S: As I am new to it. I don't know how to build the same locally
- [N ] Does your submission pass `brew audit --new-formula <formula>` (after doing `brew install <formula>`)?

---

brew install jing is not working for the present code.

I am receiving the following error:

```
==> Using the sandbox
==> Downloading https://jing-trang.googlecode.com/files/jing-20091111.zip

curl: (22) The requested URL returned error: 404 Not Found
Error: Failed to download resource "jing"
Download failed: https://jing-trang.googlecode.com/files/jing-20091111.zip
```

As the current homepage is :https://code.google.com/archive/p/jing-trang/
and url is url "https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/jing-trang/jing-20091111.zip"

Regards,
Sachin

### CODE REVIEW

File: Formula/jing.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class Jing < Formula
   desc "RELAX NG validator"
-  homepage "https://code.google.com/p/jing-trang/"
-  url "https://jing-trang.googlecode.com/files/jing-20091111.zip"
+  homepage "https://code.google.com/archive/p/jing-trang/"   #"https://code.google.com/p/jing-trang/"
```
Comment (dunn) [inline Formula/jing.rb]: If the URLs have changed, you don't need the comments; just the new values is fine.

**Comment by tdsmith** [comment]
Better yet, we can update to its new home: http://www.thaiopensource.com/relaxng/jing.html, https://github.com/relaxng/jing-trang

**Comment by tdsmith** [comment]
Also an audit failure: `  * 4: Trailing whitespace was found`

**Comment by zmwangx** [comment]
Ref https://github.com/Homebrew/homebrew-core/pull/4312.

**Comment by zmwangx** [comment]
Closing as `jing` has been deported to boneyard. #4799

**Pull request closed without merging**